src/Abp.ZeroCore.IdentityServer4/IdentityServer4/AbpProfileService.cs

```diff
@@ -38,7 +41,10 @@ public override async Task IsActiveAsync(IsActiveContext context)
             var tenantId = context.Subject.Identity.GetTenantId();
             using (_unitOfWorkManager.Current.SetTenantId(tenantId))
             {
-                await base.IsActiveAsync(context);
+                var sub = context.Subject.GetSubjectId();
+                var user = await _userManager.FindByIdAsync(sub);
+
+                context.IsActive = user != null && user.IsActive;
```

Comment:
You didn't call base.IsActiveAsync at all. Do you know what it does? It would be better first call Base.IsActive, if it's true then also check for user.IsActive.

Reply:
It checks whether user is null or not. But right, I changed.